src/cgame/cg_particles.cpp

```diff
@@ -2121,6 +2121,14 @@ Calculate the fraction of time passed
 */
 static float CG_CalculateTimeFrac( int birth, int life, int delay )
 {
+	/* Don't compute time frac if life is equal to delay
```

Comment:
This seems like a potential asset bug. It implies that the particle is configured to live for 0 time?